packages/extension-ui/src/Popup/Forget.tsx

```diff
@@ -15,6 +15,22 @@ type Props = RouteComponentProps<{ address: string }>;
 function Forget ({ match: { params: { address } } }: Props): React.ReactElement<Props> {
   const { t } = useTranslation();
   const onAction = useContext(ActionContext);
+  const { accounts } = useContext(AccountContext);
+  const isExternal = useMemo(() => {
+    let res: boolean | undefined;
+
+    accounts.some((account) => {
+      if (account.address === address) {
+        res = account.isExternal;
+
+        return true;
+      }
+
+      return false;
+    });
+
+    return res;
+  }, [accounts, address]);
```

Comment:
```suggestion
  const isExternal = useMemo(
    () => accounts.some((account) =>
      account.address === address && account.isExternal
    )),
    [accounts, address]);
```

I understand the above, mapping to address first, but it seems weird to me. If that approach is really to be followed, either filter + isExternal or find + isExternal.

Reply:
thanks, I used find.